tests/pytests/integration/daemons/test_memory_leak.py

```diff
@@ -46,6 +49,7 @@ def file_add_delete_sls(testfile_path, base_env_state_tree_root_dir):
 
 @pytest.mark.skip_on_fips_enabled_platform
 @pytest.mark.skip_on_darwin(reason="MacOS is a spawning platform, won't work")
+@pytest.mark.skipif(GITHUB_ACTIONS, reason="Test is failing in GitHub Actions")
```

Comment:
```suggestion
@pytest.mark.skip_on_env("GITHUB_ACTIONS", present=True)
```

Reply:
However, this test is not failing on our test suite...

Reply:
It seems this one is flaky and fails randomly. It feels that this happens more often when running on GH.